font_struct.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _font_struct_h
+#define _font_struct_h
+#include <stdint.h>
+/**
+ * Struct that holds a font and all relevant information
+ */
+typedef struct {
+    uint16_t bytes_per_char; // the size of the char sprite in bytes (including padding)
```

Comment:
Why are these 16-bit vs the uint8_t they were before? A character greater than 255 doesn't seem likely / useful. Granted the sprite size would be limited to ~45x45 but that doesn't feel very limiting.

Reply:
This is the size of the character sprite, so the memory used to store the bitmap of the character. If a font is large, it very well might take more than 255 bytes to store the sprite in. (The display is 128x64 -> 128 x 8 bytes -> 1024 bytes large (which may all be a single char sprite) this means we need more than 255 bytes in extreme cases. Given that the pico can do 32bit load and store and we have plenty of ram i dont think a single byte per font element would matter much.